pino.js

```diff
@@ -217,7 +224,7 @@ function fastRep (s) {
   } else {
     result += str.slice(last)
   }
-  return result
+  return point < 32 ? JSON.stringify(str) : '"' + result + '"'
```

Comment:
I meant if you moved the return statement into the loop, this return statement would be simplified. I just didn't see the early break in the for loop condition – all just a misunderstanding, my bad!